brokurly/urls.py

```diff
@@ -1,4 +1,5 @@
 from django.urls   import path, include
 
 urlpatterns = [
+    path('', include('product.urls'))
```

Comment:
```suggestion
    path('products', include('product.urls'))
```
어떤 데이터에 접근할 엔드포인트인지 알려주기 위해서 앱 이름을 경로 앞에 붙여주시는 것이 좋습니다.

Reply:
리뷰 감사합니다! 수정했습니다